js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -158,6 +158,13 @@ export const AssetGraphExplorerWithData: React.FC<WithDataProps> = ({
     ? graphQueryItems.map((a) => a.node)
     : Object.values(assetGraphData.nodes).map((a) => a.definition);
 
+  const hasLaunchPermission = React.useMemo(() => {
+    if (selectedDefinitions.length) {
+      return selectedDefinitions.some((definition) => definition.hasMaterializePermission);
+    }
```

Comment:
this should be all not some right? you need to have permissions for every asset that's being materialzed